app/controllers/results_controller.rb

```diff
@@ -13,6 +13,7 @@ class ResultsController < ApplicationController
     # required because MathJax dynamically changes
     # style. # TODO: remove this when possible
     p.style_src :self, "'unsafe-inline'"
+    p.frame_src 'https://www.youtube.com', 'https://drive.google.com', 'https://play.library.utoronto.ca'
```

Comment:
Make these a constant on the class and refer to that constant here (and in the submissions_controller.rb)